src/equations/compressible_euler_2d.jl

```diff
@@ -1959,6 +1986,31 @@ end
     return S
 end
 
+# Transformation from conservative variables u to d(s)/d(u)
+@inline function gradient_conservative(::typeof(entropy_math),
+                                       u, equations::CompressibleEulerEquations2D)
+    return cons2entropy(u, equations)
+end
+
+# Calculate specific entropy for conservative variable u
+@inline function entropy_spec(u, equations::CompressibleEulerEquations2D)
+    rho, rho_v1, rho_v2, rho_e = u
+
+    # Modified specific entropy from Guermond et al. (2019)
+    s = (rho_e - 0.5 * (rho_v1^2 + rho_v2^2) / rho) * (1 / rho)^equations.gamma
+
+    # Other specific entropy
+    # rho_sp = rho/((equations.gamma - 1.0) * (rho_e - 0.5 * rho * v_square))
+    # s = log(p) - (equaions.gamma + 1) * log(rho)
```

Comment:
We should then remove this if not used.
```suggestion
```

Reply:
Done in [ad7e1da](https://github.com/trixi-framework/Trixi.jl/pull/1792/commits/ad7e1dabf3c532d18b8af45b0b490b47b4a4e3ef)